indico/modules/events/registration/templates/management/registration_details.html

```diff
@@ -1,58 +1,44 @@
 {% extends 'layout/base.html' %}
+{% from 'events/registration/display/_registration_summary_blocks.html' import render_registration_summary,
+                                                                               render_payment_summary %}
 
 {% block title %}
     {%- trans %}Registration details{% endtrans -%}
 {% endblock %}
 
 {% block content %}
-    <dl class="i-data-list">
-        <dt>{% trans %}Registration ID{% endtrans %}</dt>
-        <dd>{{ registration.friendly_id }}</dd>
-        <dt>{% trans %}Registration date{% endtrans %}</dt>
-        <dd>{{ registration.submitted_dt | format_datetime }}</dd>
-        {% if registration.user %}
-            <dt>{% trans %}Indico user{% endtrans %}</dt>
-            <dd>{{ registration.user.full_name }}</dd>
-        {% endif %}
-    </dl>
-
-    {% set data = registration.data_by_field %}
-    {% for section in registration.registration_form.sections %}
-        <h3>{{ section.title }}</h3>
-        <dl class="i-data-list">
-            {% for field in section.children if field.is_field and (field.is_enabled or field.id in data) %}
-                <dt>{{ field.title }}</dt>
-                <dd>
-                    {% set value = data[field.id].friendly_data if field.id in data else '' %}
-                    {% if field.input_type == 'file' and value %}
-                        <a href="{{ url_for('.registration_file', data[field.id].locator.file) }}">{{ value }}</a>
```

Comment:
OK, looks like the template in the display area does not add a link to the actual file. I will add it.